core/internal/logger.hpp

```diff
@@ -0,0 +1,61 @@
+/**
+ * This is the header for the logger.
+ * Very little of the logger will be exposed through the API.
+ * Author: Jake Roggenbuck
+ * 10-08-23
+ */
+
+#pragma once
+
+#include <iostream>
+
+namespace jml {
+
+enum Severity { DEBUG, INFO, WARN, ERR, FATAL };
+
+static const std::string severity_text[] = {"DEBUG", "INFO", "WARN", "ERR",
+                                            "FATAl"};
```

Comment:
The `l` is lowercased :)
```suggestion
                                            "FATAL"};
```

Reply:
(you can click the "Commit suggestion" button to accept it)

Reply:
> The `l` is lowercased :)
> 
> ```suggestion
> 
>                                             "FATAL"};
> 
> ```

Hahahah, omg